diff --git a/node_modules/js-clipper/clipper.js b/node_modules/js-clipper/clipper.js
index cd42dec..792a0ef 100755
--- a/node_modules/js-clipper/clipper.js
+++ b/node_modules/js-clipper/clipper.js
@@ -5952,10 +5952,10 @@ ClipperLib.Clipper.DistanceSqrd = function (pt1, pt2)
 ClipperLib.Clipper.DistanceFromLineSqrd = function (pt, ln1, ln2)
 {
   //The equation of a line in general form (Ax + By + C = 0)
-  //given 2 points (x¹,y¹) & (x²,y²) is ...
-  //(y¹ - y²)x + (x² - x¹)y + (y² - y¹)x¹ - (x² - x¹)y¹ = 0
-  //A = (y¹ - y²); B = (x² - x¹); C = (y² - y¹)x¹ - (x² - x¹)y¹
-  //perpendicular distance of point (x³,y³) = (Ax³ + By³ + C)/Sqrt(A² + B²)
+  //given 2 points (x1,y1) & (x2,y2) is ...
+  //(y1 - y2)x + (x2 - x1)y + (y2 - y1)x1 - (x2 - x1)y1 = 0
+  //A = (y1 - y2); B = (x2 - x1); C = (y2 - y1)x1 - (x2 - x1)y1
+  //perpendicular distance of point (x3,y3) = (Ax3 + By3 + C)/Sqrt(A2 + B2)
   //see http://en.wikipedia.org/wiki/Perpendicular_distance
   var A = ln1.Y - ln2.Y;
   var B = ln2.X - ln1.X;
